src/scheduler/pbs_sched_utils.cpp

```diff
@@ -623,19 +631,25 @@ connect_svrpool()
 			log_errf(pbs_errno, __func__, "Scheduler %s could not connect with all the configured servers", sc_name);
 			sleep(2);
 			close_servers();
-			continue;
+			goto CONTINUE;
 		}
 
 		if (pbs_register_sched(sc_name, clust_primary_sock, clust_secondary_sock) != 0) {
 			log_errf(pbs_errno, __func__, "Couldn't register the scheduler %s with the configured servers", sc_name);
 			/* wait for 2s for not to burn too much CPU, and then retry connection */
 			sleep(2);
 			close_servers();
-			continue;
+			goto CONTINUE;
 		}
 
 		/* Reached here means everything is success, so we will break out of the loop */
+BREAK:		if (sigprocmask(SIG_SETMASK, &prevsigs, NULL) == -1)
```

Comment:
Put the code below the label.  Also labels are usually lower case.  I'd rename them and use a lower case label.  How about unmask_break and unmask_continue